kubernetes-ts-multicloud/index.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright 2016-2019, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as k8s from "@pulumi/kubernetes";
+import * as pulumi from "@pulumi/pulumi";
+import * as aks from "./aks";
+import * as app from "./app";
+import * as eks from "./eks";
+import * as gke from "./gke";
+import * as local from "./local";
+
+// Create Kubernetes clusters.
+// Note: Uncomment lines for the desired cloud providers.
+// const aksCluster = new aks.AksCluster("multicloud", {});
+// const eksCluster = new eks.EksCluster("multicloud", {});
+// const gkeCluster = new gke.GkeCluster("multicloud", {});
```

Comment:
Why not keep all clusters in the example by default, and let users comment out the ones they don't need?